clients/tests/rocsparse_test_main.cpp

```diff
@@ -173,6 +174,74 @@ class ConfigurableEventListener : public testing::TestEventListener
 
 int main(int argc, char** argv)
 {
+    //
+    // Enable debug mode for testing.
+    //
+    rocsparse_enable_debug();
+
+    //
+    // Enable test debug arguments.
+    //
+    if(rocsparse_clients_envariables::is_defined(
```

Comment:
Ok this is what turns on checking the signature arguments?

Reply:
No.
If no environment variable tells us how to define the test debug argument mode, then we enable it by default.